src/sonic-yang-models/yang-models/sonic-vlan.yang

```diff
@@ -158,6 +158,10 @@ module sonic-vlan {
 						(contains(../ip-prefix, '.') and current()='IPv4')";
 					type stypes:ip-family;
 				}
+
+				leaf secondary	{
```

Comment:
Please add a description for this field.

Reply:
Added